pkg/db/queries.sql

```diff
@@ -66,3 +66,12 @@ LIMIT @num_rows;
 DELETE FROM staged_originator_envelopes
 WHERE id = @id;
 
+-- name: SelectVectorClock :many
+SELECT
+	originator_node_id,
+	max(originator_sequence_id)::BIGINT AS originator_sequence_id
```

Comment:
With the right indexes this query should be fast. We'll want to do some `EXPLAIN ANALYZE`ing on it at some point to make sure we are taking advantage of the indexes here.

Reply:
The primary key on this table is `originator_node_id, originator_sequence_id`, which I think is a b-tree index, so hopefully it'll be fast - but would love to get some analyzing done